x-pack/plugins/actions/common/connector_feature_config.ts

```diff
@@ -34,6 +35,13 @@ const compatibilityGenerativeAI = i18n.translate(
   }
 );
 
+const compatibilityGenerativeAIForObservability = i18n.translate(
+  'xpack.actions.availableConnectorFeatures.compatibility.generativeAIForObservability',
+  {
+    defaultMessage: 'Generative AI For Observability',
```

Comment:
Have we put any thoughts on removing “Generative AI” feature and creating the same fine grain feature for security solution?
- ~~Generative AI~~
- Generative AI For Observability
- Generative AI For Security Solution

I wonder if that will be more descriptive and consistent on which assistant a connector supports? @stephmilovic @shanisagiv1 any thoughts?